src/System.Drawing.Common/src/System/Drawing/GdiplusNative.Unix.cs

```diff
@@ -54,29 +54,20 @@ private static void PlatformInitialize()
 
             private static void InitializeSystemContext()
             {
-                if (Environment.GetEnvironmentVariable("not_supported_MONO_MWF_USE_NEW_X11_BACKEND") != null || Environment.GetEnvironmentVariable("MONO_MWF_MAC_FORCE_X11") != null)
+                if (Environment.GetEnvironmentVariable("SYSTEM_DRAWING_COMMON_FORCE_X11") != null)
                 {
                     UseX11Drawable = true;
                 }
                 else
                 {
-                    IntPtr buf = Marshal.AllocHGlobal(8192);
-                    // This is kind of a hack but gets us sysname from uname (struct utsname *name) on
-                    // linux and darwin
-                    if (uname(buf) != 0)
+                    if (RuntimeInformation.IsOSPlatform(OSPlatform.OSX))
                     {
-                        // WTH: We couldn't detect the OS; lets default to X11
-                        UseX11Drawable = true;
+                        UseCarbonDrawable = true;
                     }
                     else
                     {
-                        string os = Marshal.PtrToStringAnsi(buf);
-                        if (os == "Darwin")
-                            UseCarbonDrawable = true;
-                        else
-                            UseX11Drawable = true;
+                        UseX11Drawable = true;
```

Comment:
Seems like it could also be simplified to:
```C#
if (Environment.GetEnvironmentVariable("SYSTEM_DRAWING_COMMON_FORCE_X11") != null ||
    !RuntimeInformation.IsOSPlatform(OSPlatform.OSX))
{
    UseX11Drawable = true;
}
else
{
    UseCarbonDrawable = true;
}
```
Also, since it looks like it's setting one and only one of these to true, is there are a reason there are two variables?  Why isn't it just UseCarbonDrawable == true means carbon and == false means X11?  In which case it would just be:
```C#
UseCarbonDrawable =
    Environment.GetEnvironmentVariable("SYSTEM_DRAWING_COMMON_FORCE_X11") != null ||
    !RuntimeInformation.IsOSPlatform(OSPlatform.OSX);
```